extension/src/popup/views/AccountCreator/index.tsx

```diff
@@ -131,12 +131,12 @@ export const AccountCreator = () => {
                         />
                       )}
                     </Field>
-                  </FormRow>
+                  </div>
                   {/* TODO - add error to Checkbox in SDS */}
-                  {errors.termsOfUse && touched.termsOfUse
-                    ? errors.termsOfUse
-                    : null}
-                  <div className="AccountCreator__button-row">
+                  {touched.termsOfUse ? (
```

Comment:
❓  are we still adding the error field to the Checkbox in SDS? Or does adding a <FormError> work (in which case can you remove the TODO a couple lines up?)

Reply:
I would still like to move this to SDS. We needed <FormError> in a couple of other non-checkbox places, so I figured I should keep this aligned until we're able to make that change in SDS